src/libs/utils.js

```diff
@@ -129,7 +129,6 @@ export const isEmpty = obj => {
   if (obj !== null) {
     Object.keys(obj).forEach(key => {
```

Comment:
I wonder if we can clean this function up a bit. Let's chat.

Reply:
eslint complain the use of for loop on array, keeping forEach. moved the null check in advanced.